app/javascript/controllers/new_message_controller.js

```diff
@@ -1,5 +1,3 @@
-"use strict";
-
```

Comment:
Based on my reading, ES6 modules are always in strict mode so we shouldn't need this